library/network/src/modules/SuSEFirewall.rb

```diff
@@ -42,6 +44,14 @@ def self.create
   end
 
   # ----------------------------------------------------------------------------
+  # SuSEFirewalld Class. Trying to provide relevent pieces of SF2 functionality via
```

Comment:
@mvidner that's fine but that requires to rewrite all 54 commits so it will take a while